### PR TITLE
Fix bug with displaying empty suggetsions

### PR DESCRIPTION
**Pull request by BlakeDonn**
#### What's this PR do?  

- Fixes bug where app would crash if no activity was filtered
- Fixes naming for test

#### How should this be manually tested?   
 
- Check for any app-crashing bugs

#### Any background context you want to provide?  

- This brings our application to a near-complete state

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -235,7 +235,7 @@ class App extends Component {
           ></Route>
           <Route
             exact 
-            path='/:activity/:results'
+            path={this.state.randomActivity ? '/:activity/:results': '/form'}
```
Comment (Atos20) [inline src/Components/App/App.js:238]: I think this fixes part of the bug but we should definitely come back and do a more thorough error handling.

File: src/Components/ResultPage/ResultPage.test.js [hunk 2]
```diff
@@ -108,7 +108,7 @@ describe('ResultPage', () => {
         
       });
 
-    it('It sshould display a "back button" when an error is found', () => {
+    it('It should display a "back button" when an error is found', () => {
```
Comment (Atos20) [inline src/Components/ResultPage/ResultPage.test.js:111]: thanks 👌

**Review by Atos20 — APPROVED** [review]
Great addition!

**Review by nicolegooden — APPROVED** [review]
Looks good to me.  Solid effort toward the bug fix!

**Pull request merged**